lib/actions/user.js

```diff
@@ -181,3 +188,91 @@ export function deleteUserMonitoredTrip (id) {
     }
   }
 }
+
+/**
+ * Requests a verification code for the logged-in user.
+ */
+export function requestPhoneVerificationCode (originalUserData, newPhoneNumber) {
+  return async function (dispatch, getState) {
+    const { otp, user } = getState()
+    const { otp_middleware: otpMiddleware = null } = otp.config.persistence
+
+    if (otpMiddleware) {
+      const { accessToken } = user
+
+      // FIXME: (required by middleware) Only temporarily update the user's phone number
+      //   in the database with the pending one,
+      //   and cache the previous value and verification state so they can be reverted.
+      //   The full user record will be updated upon the user clicking Finish/Save preferences.
+      // TODO: Figure out what should happen if the user refreshes the browser at this stage.
+
+      // Make a clone of the original userData object.
+      const userData = clone(originalUserData)
```

Comment:
Might be more readable to rename `userData` to `newUserData`.

Reply:
Fixed in https://github.com/opentripplanner/otp-react-redux/pull/224/commits/626fdb7a50e90c7a827c98e3d9a00d29f2025365